vendor/k8s.io/kubernetes/staging/src/k8s.io/client-go/scale/client.go

```diff
@@ -197,7 +197,10 @@ func (c *namespacedScaleClient) Update(resource schema.GroupResource, scale *aut
 		Body(scaleUpdateBytes).
 		Do()
 	if err := result.Error(); err != nil {
-		return nil, fmt.Errorf("could not update the scale for %s %s: %v", resource.String(), scale.Name, err)
```

Comment:
@deads2k @tnozicka isn't this "just enough" fix to stop the flakiness of scaler and leave us with rest in 1.10 world?

Reply:
It might, I have been looking from where that would come from

Reply:
How did this get in? I though the clients are generated...

Reply:
@sttts ^^

Reply:
> How did this get in? I though the clients are generated...

The scale client is a hand-rolled, generic client .


I would not pick this by itself.